pkg/appgw/internaltypes_test.go

```diff
@@ -107,25 +109,25 @@ var _ = Describe("Test string key generators", func() {
 			Expect(len(actual)).To(Equal(80))
 		})
 		It("generateProbeName preserves keys in 80 charaters of length or less", func() {
-			expected := "k8s-ag-ingress-xxxxxx-yyyyyy-pb-zzzz"
+			expected := "xxxxxx-yyyyyy-pb-zzzz"
 			serviceName := "xxxxxx"
 			servicePort := "yyyyyy"
 			ingress := "zzzz"
 			actual := generateProbeName(serviceName, servicePort, ingress)
 			Expect(actual).To(Equal(expected))
 		})
 		It("generateProbeName relies on governor and hashes long keys", func() {
-			expected := "k8s-ag-ingress-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-a68359d4111b72692502fe32a108c47f"
+			expected := "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-yyyy-f6ec0d78e6dd6290971a7de1ed28e6a1"
```

Comment:
append `agPrefix +` ?

Reply:
I deliberately removed `agPrefix + ` from this test.
The expected value contains an MD5 hash, since we are testing >80 char string.
The MD5 hash would change based on the agPrefix, so this test won't automatically adapt to a code change where we again change the default value.
In other words - if one day we change default for agPrefix, we would have to come here and change the MD5 hash.
So I thought I'd leave it out since the new default is blank.

Reply:
Cool!